### PR TITLE
Link preview fixes

### PR DESCRIPTION
**Pull request by danarish4**
Click below to open examples:
rich-content: https://rich-content-linkPreviewFixes.surge.sh/
rich-content-storybook: https://rich-content-storybook-linkPreviewFixes.surge.sh/

### CODE REVIEW

**Comment by oded-soffrin** [comment]
does the link should open in new window ?

File: packages/plugin-link-preview/web/src/consts.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
   size: 'content',
+  link: {
+    target: '_blank',
+    rel: 'noopener',
+  },
 };
 
 export const REMOVE_LINK_PREVIEW = 'remove-link-preview';
+
+export const AUTO_GENERATED_LINK_PREVIEW_PROVIDER = 'wix';
```
Comment (shaulgo) [inline packages/plugin-link-preview/web/src/consts.js:12]: we try to keep wix out of our project. What is this for?

Comment (oded-soffrin) [reply packages/plugin-link-preview/web/src/consts.js:12]: That's validation on the service response data, meant to filter out auto-generated HTML by the service. 
but this is a valid point, the filter logic should be moved here
 https://github.com/wix-private/rich-content-services/blob/master/wix-rich-content/src/linkPreviewStrategy/linkPreviewStrategy.ts

**Comment by oded-soffrin** [comment]
merging and add to tasks to remove the wix-referer in regard to  data validation

**Pull request merged**